crates/rspack_plugin_javascript/src/dependency/context/mod.rs

```diff
@@ -37,9 +38,35 @@ fn create_resource_identifier_for_context_dependency(
     .map(|x| x.to_source_string())
     .unwrap_or_default();
   let mode = options.mode.as_str();
-  // TODO: need `RawChunkGroupOptions`
+  let referenced_exports = options
+    .referenced_exports
+    .as_ref()
+    .map(|x| x.iter().map(|x| format!(r#""{x}""#)).join(","))
+    .unwrap_or_default();
+  let mut group_options = String::new();
+
+  if let Some(GroupOptions::ChunkGroup(group)) = &options.group_options {
+    if let Some(chunk_name) = &group.name {
+      group_options += chunk_name;
+    }
+    group_options += "{";
+    if let Some(o) = group.prefetch_order {
+      group_options += "prefetchOrder: ";
```

Comment:
Perhaps need to add `,` as the separator of the fields of `groupOptions`, can be merged after fixing this.